### ISSUE TITLE
Changing SpineData/Texture of Spine object?

### ISSUE THREAD

**Issue by Shadowstep33** [pixijs/pixi-spine#63]
Hi, long time no chat :) 

I was curious if there is a way to change what Spine atlas is associated with the Spine object. Essentially, just like I can do .texture = new Texture(); for Sprites, I want to know how to do that for Spines also

**Comment by ivanpopelyshev** [comment]
Hello!

At the moment, there are two ways:

1) Do the parsing one more time. Look at (Pixi.Spine without loader)[https://github.com/pixijs/pixi-spine/issues/62]

2) Use spine skins engine.

**Comment by ivanpopelyshev** [comment]
Try parse skeleton several times with different atlases , and change 

```javascript
spine.skeleton = skeleton2;
```

**Comment by ivanpopelyshev** [comment]
Its not possible to change it at runtime, due to problems like "what if spine object uses multiple atlases".

Do you really think that you need that feature?

**Comment by Shadowstep33** [comment]
I tried changing some of the properties such as skeleton, stateData, etc. but I changed them all at once. I will try changing just the skeleton.

The feature isn't needed, but would make making cool transitions easier :) For example, I am trying to distort one monster and have it turn into another monster using brightness and a displacement map. If I could change the "texture(/skeleton)" of the sprite mid animation, it would be very easy to do the animation. As it is now, to perform it "smootly" I would need two Spines overlayed animating at the same time (in which case I generally end up having to keep a reference to it until I am finished animating it)
